### DISCUSSION TITLE
User Experience (Feedback)

### DISCUSSION THREAD

**Discussion by voodoohop** [pollinations/pollinations#79] [Pollinations]
from Caroline Barrueco:

- [x] refer to instructions not help
- [x] return to this (instead of prior) browser tab
- [x] Sometimes one has to go to the Colab tab and click "Runtime -> Run All". (Rewrite to repeat step 3 or better words)
- [x] during heavy usage. Clarify
- [x] need links to the social media profiles
- [ ] first page: what style would you like to create

**Comment by voodoohop** [comment]
from @ElliotEtag
- [ ]  notebook description should be more in the "background" (darker color or expandable and collapsed initially)?
- [ ] Shorter text that points out the particularity of this model
- [ ] Icon in notebook list strange
- [ ] Notebook list as a grid
- [ ] Menu comes in from the left
- [ ] social media icons and get help / contribute message fixed bottom floating
- [ ] better progress
- [ ] cancel button
- [ ] more obvious that its calculating

**Comment by gokaykucuk** [comment]
- [ ] Explain what does GPU mean, and maybe the potential options user has with a mouseover questionmark next to GPU.

**Comment by voodoohop** [comment]
We could get inspiration from: https://observablehq.com/

**Comment by voodoohop** [comment]
Use AI/machine learning art in the first text

the word "notebook" is not clear to a general audience

**Reply by voodoohop** [reply]
Combine bullet point list on front page to a more evocative sentence? @ale-rls

**Comment by Juanoto2012** [comment]
### Question about the API usage and future pricing
**Hi Team! I'm a huge fan of Pollinations.ai and I'm currently using it for a project called Ventarys. I was wondering: do you plan to keep the API free and unlimited in the long term? I'd love to know if there are any upcoming changes to the usage limits or if a paid tier is coming soon so I can plan my project accordingly. Thanks for the amazing work!**

**Reply by Devestatio** [reply]
I don't believe the API is completely Unlimited, it's just that the current rates are very generous, and allow you to do quite a lot per day.  If I am wrong please do let's us know, I do imagine though that there will be higher pricing in the future, and as more users discover and begin to use the service.

~Edit~
NM, you are correct the Private API key is currently 100% free, the Public key is not. But I am sure that will change eventually.

**Comment by killuaX900ping** [comment]
Im gay

**Reply by Juanoto2012** [reply]
...

**Comment by relaxroom066-star** [comment]
Currently, login requires a GitHub or Discord account. While this works great for developers, it creates a high barrier to entry for artists, writers, and non-technical users who want to use the platform but don't code. Adding a standard Google Sign-in option would open up the tool to a much wider audience and reduce onboarding friction

**Reply by Juanoto2012** [reply]
Okay, then they'll add Google.?

**Comment by Juanoto2012** [comment]
How the secret key work? , Is good for production apps?

**Reply by Juanoto2012** [reply]
I love byop system

**Reply by Juanoto2012** [reply]
I migrated from pollinations to perchance for the Ventarys AI project .

**Comment by infometadatagenerator-art** [comment]
Much better than previous. But now missing anonymous api call which isn't good

**Reply by Juanoto2012** [reply]
Yeah
